docs/content/docs/dev/table/sqlClient.md

```diff
@@ -169,559 +169,394 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
 
 ```text
 [INFO] Table update statement has been successfully submitted to the cluster:
 Cluster ID: StandaloneClusterId
 Job ID: 6f922fe5cba87406ff23ae4a7bb79044
-Web interface: http://localhost:8081
 ```
 
 <span class="label label-danger">Attention</span> The SQL Client does not track the status of the running Flink job after submission. The CLI process can be shutdown after the submission without affecting the detached query. Flink's `restart strategy` takes care of the fault-tolerance. A query can be cancelled using Flink's web interface, command-line, or REST API.
 
-{{< top >}}
+### Execute a batch of SQL statements
 
-SQL Views
----------
+Sometimes it's prefer to batch insert statements and optimize/execute the batched statements together. In SQL Client, it supports to parse and execute statement set in SQL.
 
-Views allow to define virtual tables from SQL queries. The view definition is parsed and validated immediately. However, the actual execution happens when the view is accessed during the submission of a general `INSERT INTO` or `SELECT` statement.
+```text
+Flink SQL> BEGIN STATEMENT SET;
+[INFO] Begin a statement set.
 
-Views can either be defined in [environment files](sqlClient.html#environment-files) or within the CLI session.
+Flink SQL> INSERT INTO MySink1 SELECT * FROM MySource1;
+[INFO] Add SQL update statement to the statement set.
 
-The following example shows how to define multiple views in a file. The views are registered in the order in which they are defined in the environment file. Reference chains such as _view A depends on view B depends on view C_ are supported.
+Flink SQL> INSERT INTO MySink2 SELECT * FROM MySource1;
+[INFO] Add SQL update statement to the statement set.
 
-```yaml
-tables:
-  - name: MyTableSource
-    # ...
-  - name: MyRestrictedView
-    type: view
-    query: "SELECT MyField2 FROM MyTableSource"
-  - name: MyComplexView
-    type: view
-    query: >
-      SELECT MyField2 + 42, CAST(MyField1 AS VARCHAR)
-      FROM MyTableSource
-      WHERE MyField2 > 200
+Flink SQL> END;
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] Execute statement in sync mode. Please wait for the execution finish...
+[INFO] Complete execution of the SQL update statement.
 ```
 
-Similar to table sources and sinks, views defined in a session environment file have highest precedence.
+<span class="label label-danger">Attention</span> It's allowed to add `INSERT` statements into the statement set only.
 
-Views can also be created within a CLI session using the `CREATE VIEW` statement:
+### Execute DML Statements one by one
 
-```text
-CREATE VIEW MyNewView AS SELECT MyField2 FROM MyTableSource;
-```
-
-Views created within a CLI session can also be removed again using the `DROP VIEW` statement:
+Sometimes it's better to execute statements one by one. In SQL Client, it can use the option `table.dml-sync` to wait for the job finishes.
```

Comment:
Suggestion:

By default, SQL Client executes DML statements asynchronously. That means, SQL Client will submit a job for the DML statement to a Flink cluster, and not wait for the job to be finished. So that SQL Client can submit multiple jobs at the same time. This can be useful in streaming mode, because streaming mode is long running. 

However, for batch users, it's more common that the next DML statement requires to wait for the previous DML statement to be finished. In order to execute DML statements synchronously, you can set `table.dml-sync` option to true in SQL Client.